website/source/guides/operating-a-job/advanced-scheduling/preemption-service-batch.html.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: "guides"
+page_title: "Preemption (Service and Batch Jobs)"
+sidebar_current: "guides-operating-a-job-preemption-service-batch"
+description: |-
+  The following guide walks the user through enabling and using preemption on
+  service and batch jobs in Nomad Enterprise (0.9.3 and above). 
+---
+
+# Preemption for Service and Batch Jobs
+
+~> **Enterprise Only!** This functionality only exists in Nomad Enterprise. This
+is not present in the open source version of Nomad.
+
+Starting with Nomad 0.9, every job in Nomad has a [priority][priority]
+associated with it. Preemption allows Nomad to evict current allocations with
+newly scheduled allocations that have been assigned a higher priority. The
+evicted allocation is temporarily displaced until the cluster has the capacity
```

Comment:
Jobs whose allocations are blocked, temporarily go into "pending" status until the cluster has additional capacity to run it.